README.md

```diff
@@ -75,6 +75,18 @@ npm run build:fast
 
 See also: [Contributing](#Contributing)
 
+### Other First-Time Build Requirements
+
+- [node-gyp](https://github.com/nodejs/node-gyp) Requirements
+  - Ensure that the requirements listed [here](https://github.com/nodejs/node-gyp#installation) are satisfied for your environment.
+  - Also note that the version of `node-gyp` pulled in by our current version of `Node.js` (14.x.x) **is not** compatible with Visual Studio 2022.
+    This has been fixed on more recent versions of `node-gyp`, but until we update to a more recent version of `Node.js`, we recommend using an earlier version of Visual Studio / its build tools.
+    Version 17 seems to be stable.
```

Comment:
This is Windows specific. Seems like if we want to have a section just for windows, we might want to put this there. Maybe also note that you can avoid windows specific issues by building in WSL?

Reply:
Good call to both points :) Will update.

Reply:
Updated